### PR TITLE
Add Documentation for Installing and Running qt-gui

### PR DESCRIPTION
**Pull request by iandonnelly**

### CODE REVIEW

**Comment by easyxtarget** [comment]
I thought I submitted this a while ago but apparently nit.

File: src/tools/qt-gui/README.md [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
-    cmake -DTOOLS=ALL -DCMAKE_PREFIX_PATH=/opt/Qt5.3.0/5.3/gcc_64/lib/cmake .
+This document aims to provide information about Elektra's new `qt-gui` feature. `qt-gui` offers a graphical
+interface for users of Elektra. It allows users to create, manage, edit, and delete keys stored in KDB.
 
+## Compiling and Installation ##
 
-Dependencies: Qt5.3 and libdrm-dev
+### Dependencies ###
 
-If you have wheezy-backports enabled and do not use kde4, you can also install:
```
Comment (markus2330) [inline src/tools/qt-gui/README.md]: please state that qtdeclarative5-dev is in wheezy-backports

File: src/tools/qt-gui/README.md [hunk 2]
```diff
@@ -1,14 +1,35 @@
...
+In order to compile and use the new `qt-gui` there are a few dependencies which must be installed. 
 
-    apt-get install qtdeclarative5-dev
+- qt5.3 or greater
+- libdrm-dev
+
+I was able to install the correct dependencies on KDE using the command:
```
Comment (markus2330) [inline src/tools/qt-gui/README.md]: Please state your system (distribution, version, ...)

File: src/tools/qt-gui/README.md [hunk 3]
```diff
@@ -1,14 +1,35 @@
...
+In order to compile and use the new `qt-gui` there are a few dependencies which must be installed. 
 
-    apt-get install qtdeclarative5-dev
+- qt5.3 or greater
+- libdrm-dev
+
+I was able to install the correct dependencies on KDE using the command:	
+`sudo apt-get install qt5-default qml-module-qtquick-controls qml-module-qtquick-dialogs qml-module-qtquick-layouts qml-module-qtgraphicaleffects libdrm-dev`
+
+Note: If you install qt5 manually, make sure the qt5 libarires are inlcude in `LD_LIBRARY`.
```
Comment (markus2330) [inline src/tools/qt-gui/README.md]: typos "libarires inlcude"

I think more -DCMAKE_PREFIX_PATH=/opt/Qt5.3.0/5.3/gcc_64/lib/cmake  is the important piece. I dont think LD_LIBRARY_PATH is necessary, you mistyped it and it still worked ;)

File: src/tools/qt-gui/README.md [hunk 4]
```diff
@@ -1,14 +1,35 @@
...
+
+### Compiling ###
+Compile Elektra as normal as per the [COMPILE document](COMPILE.md) making sure to include the `qt-gui` tool using the `DPLUGINS` flag.
+
+For instance:	
+`-DPLUGINS=ALL` or `-DPLUGINS=qt-gui`
```
Comment (markus2330) [inline src/tools/qt-gui/README.md]: its not PLUGINS but TOOLS

File: src/tools/qt-gui/README.md [hunk 5]
```diff
@@ -1,14 +1,35 @@
...
+
+### Installing ###
+
+To install `qt-gui` just run `sudo make install` after running CMAKE.
```
Comment (markus2330) [inline src/tools/qt-gui/README.md]: Thats not a good tip in general, please refer to doc/INSTALL.md document.

File: src/tools/qt-gui/README.md (hunk 1 above)
Comment (easyxtarget) [inline src/tools/qt-gui/README.md]: Done

File: src/tools/qt-gui/README.md (hunk 2 above)
Comment (easyxtarget) [inline src/tools/qt-gui/README.md]: Done

File: src/tools/qt-gui/README.md (hunk 3 above)
Comment (easyxtarget) [inline src/tools/qt-gui/README.md]: I added that either could be done, I had issues without adding the qt libraries to LD_LIBRARY_PATH when trying to run qt-gui

File: src/tools/qt-gui/README.md (hunk 4 above)
Comment (easyxtarget) [inline src/tools/qt-gui/README.md]: Whoops, don't know how I managed that, fixed.

File: src/tools/qt-gui/README.md (hunk 5 above)
Comment (easyxtarget) [inline src/tools/qt-gui/README.md]: Done.

**Comment by iandonnelly** [comment]
Any idea why the build failed? It can't be related to my commits, can it? 

What needs to be done so we can merge this?

**Comment by manuelm** [comment]
start build

**Pull request merged**